core/broadleaf-framework/src/main/java/org/broadleafcommerce/core/order/service/call/AddToCartItems.java

```diff
@@ -28,32 +28,32 @@
     @SuppressWarnings("unchecked")
 
     //TOOD: this should probably be refactored to be called "rows" like in other model objects
-    private List<AddToCartItem> addToCartItems =   LazyList.decorate(
-            new ArrayList<AddToCartItem>(),
-            FactoryUtils.instantiateFactory(AddToCartItem.class));
+    private List<ConfigurableOrderItemRequest> addToCartItems =   LazyList.decorate(
+            new ArrayList<ConfigurableOrderItemRequest>(),
+            FactoryUtils.instantiateFactory(ConfigurableOrderItemRequest.class));
```

Comment:
These should be OrderItemRequestDTO instead. No need to make it more specific when it doesn't need to be